test/createFunction/FunctionTesterBase.ts

```diff
@@ -62,6 +62,7 @@ export abstract class FunctionTesterBase implements vscode.Disposable {
     public async testCreateFunction(templateName: string, ...inputs: (string | undefined)[]): Promise<void> {
         await this.testCreateFunctionInternal(latestTemplates, this.funcPortalTestFolder, templateName, inputs.slice());
         await this.testCreateFunctionInternal(backupTemplates, this.backupTestFolder, templateName, inputs.slice());
+        await this.testCreateFunctionInternal(stagingTemplates, this.funcStagingPortalTestFolder, templateName, inputs.slice());
```

Comment:
nit: Do we need to call `inputs.slice()` on 3 separate occasions?

Reply:
`inputs.slice()` is an easy way to create a copy of the array. The tests will call `pop` on each array to make sure the inputs are actually used so we need three separate copies